src/qibo/models/circuit.py

```diff
@@ -127,9 +127,18 @@ class Circuit:
         nglobal (int): Base two logarithm of the number of devices. Defaults to `None`.
         nlocal (int): Total number of available qubits in each device. Defaults to `None`.
         queues (DistributedQueues): Gate queues for each accelerator device. Defaults to `None`.
+        initialize (str): Basis to define the circuit inizial state.
+        eigenstate (str): Eigenstates that will belong to the basis.
     """
 
-    def __init__(self, nqubits, accelerators=None, density_matrix=False):
+    def __init__(
+        self,
+        nqubits,
+        accelerators=None,
+        density_matrix=False,
+        initialize="Z",
```

Comment:
You could also use `basis` instead of `initialize` if I understood correctly from #720. Or is there a particular reason on why you didn't use `basis`?

Reply:
No particular reason, I think `basis` is better.